README.md

```diff
@@ -1,104 +1,216 @@
-# ntStat
+ntStat is a toolkit that uses Bloom filters to track both *k*-mer count and depth information for use in downstream applications. ntStat also models the *k*-mer count histogram using a mixture model, and infers valuable insights about the genome, sequencing data, and individual *k*-mers, *de novo*.
+
+**Contents**
+- [Installation](#installation)
+  - [From source](#from-source)
+- [Dependencies](#dependencies)
+  - [General dependencies](#general-dependencies)
+  - [C++ libraries](#c-libraries)
+  - [Python packages](#python-packages)
+- [Modules](#modules)
+  - [`filter`](#filter)
+  - [`query`](#query)
+  - [`rarity`](#rarity)
+  - [`hist`](#hist)
+- [Credits](#credits)
+- [License](#license)
+
+# Installation
+
+## From source
+
+First, ensure all [Dependencies](#dependencies) are available in the setup environment. Then, install ntStat using `meson`:
+
+```shell
+git clone https://github.com/bcgsc/ntStat.git
```

Comment:
Not a big deal, but sometimes we suggest downloading the release tarball instead of git cloning - just so a person uses a released version, plus we can track those downloads :)

Reply:
Addressed in 2057c6844555100d86789b492ba3d2a316aa628a